openpype/hosts/unreal/__init__.py

```diff
@@ -1,15 +1,19 @@
 import os
 import openpype.hosts
+from openpype.lib.applications import Application
 
 
-def add_implementation_envs(env, _app):
+def add_implementation_envs(env: dict, _app: Application) -> None:
     """Modify environments to contain all required for implementation."""
     # Set OPENPYPE_UNREAL_PLUGIN required for Unreal implementation
+
+    ue_plugin = "UE_5.0" if _app.name[:1] == "5" else "UE_4.7"
```

Comment:
It is already merged but I would recommend to add separated input for Unreal where you would tell which version of unreal it is... Using a name which is defined by admins is not best approach. If they'd want use name `testing`, you'd be lost.